Schweizer-Messer/sm_property_tree/CMakeLists.txt

```diff
@@ -29,37 +17,67 @@ add_library(${PROJECT_NAME}
   src/BoostPropertyTreeSupport.cpp
 )
 
-target_link_libraries(${PROJECT_NAME} 
-                      ${catkin_LIBRARIES} ${Boost_LIBRARIES} pthread)
+target_include_directories(${PROJECT_NAME}
+PUBLIC
```

Comment:
@frneer indentation here is still different than elsewhere, but nevermind, we can lint in a follow-up.